views/marts/product/dim_daily_license.view.lkml

```diff
@@ -120,6 +120,197 @@ view: dim_daily_license {
     description: "Whether the license also appears in Salesforce."
   }
 
+  dimension: is_feature_advanced_logging_enabled {
+    type: yesno
+    sql: ${TABLE}.is_feature_advanced_logging_enabled ;;
+    label: "Is advanced logging feature enabled?"
+    view_label: "Metadata: Data Source Info"
```

Comment:
Perhaps use a different view label for all these items. Something like `License: Feature configuration` looks like a better match.